include/common.sh

```diff
@@ -1143,4 +1151,45 @@ checkWritable()
   fi
 }
 
+generateUUID() 
+{
+  date_part=$(date +%s)
+  random_part=$((RANDOM))
+  uuid="$date_part-$random_part"
+  echo $uuid
```

Comment:
how does this uuid relate to the one from jq above? Is the uuid from above this value that gets stored into the json?